worksheet.rb

```diff
@@ -31,4 +31,136 @@
 # - the total amount of money each driver has made
 # - the average rating for each driver
 # - Which driver made the most money?
-# - Which driver has the highest average rating?
\ No newline at end of file
+# - Which driver has the highest average rating?
+
+ride_log = { DR0004: [["3rd Feb 2016",5,"RD0022",5],
+                      ["4th Feb 2016",10,"RD0022",4],
+                      ["5th Feb 2016",20,"RD0073",5],
+                      ["5th Feb 2016",45,"RD0003",2]],
+
+             DR0001: [["3rd Feb 2016",10,"RD0003",3],
+                      ["3rd Feb 2016",30,"RD0015",4],
+                      ["5th Feb 2016",45,"RD0003",2]],
+
+             DR0002: [["3rd Feb 2016",25,"RD0073",5],
+                      ["4th Feb 2016",15,"RD0013",1],
+                      ["5th Feb 2016",35,"RD0066",3]],
+
+             DR0003: [["4th Feb 2016",5,"RD0066",5],
+                      ["5th Feb 2016",50,"RD0003",2]]
+}
+
+ride_log = ride_log.sort.to_h #orders & assigns according to key
+
+def sum(array)
+  array.inject(0){|sum,x| sum + x }
+end
+
+def to_dollars(num)
+  "$#{'%.2f' % num}"
+end
+
+def rides_each_driver(log)
```

Comment:
Great work breaking up the functionality into different methods and giving each method a meaningful name. Consider also giving all variables meaningful names to increase readability. Consider what you're storing in `big_array` and what would be an appropriate name.